server/service/src/processors/transfer/requisition/mod.rs

```diff
@@ -59,6 +61,55 @@ pub(crate) enum ProcessRequisitionTransfersError {
     NameIsNotAnActiveStore(ChangelogRow),
 }
 
+fn log_system_error(
+    connection: &StorageConnection,
+    error: &ProcessRequisitionTransfersError,
+) -> Result<(), ProcessRequisitionTransfersError> {
+    let error_message = format!("ProcessRequisitionTransfersError: {:?}", error);
+    log::error!("{}", error_message);
+    system_log_entry(connection, SystemLogType::ProcessorError, &error_message)
+        .map_err(|e| ProcessRequisitionTransfersError::DatabaseError(RepositoryError::from(e)))?;
+    Ok(())
```

Comment:
Btw `.map_err(ProcessRequisitionTransfersError::DatabaseError)` would work, we should have implemented 'From' for database error though earlier on

Reply:
https://github.com/msupply-foundation/open-msupply/compare/5098-a-single-processor-error-blocks-all-future-processor-events...5098-a-single-processor-error-blocks-all-future-processor-events(Use-repository-errro)